tests/structured_conf/data/attr_classes.py

```diff
@@ -41,11 +41,26 @@ class UserDict:
     dict: Dict[str, User] = MISSING
 
 
+@attr.s(auto_attribs=True)
+class UserWithDefaultName(User):
+    name: str = "bob"
+
+
 @attr.s(auto_attribs=True)
 class MissingUserField:
     user: User = MISSING
 
 
+@attr.s(auto_attribs=True)
+class MissingUserWithDefaultNameField:
+    user: UserWithDefaultName = MISSING
```

Comment:
Ditto:
merge with `User(name=MISSING)`

Reply:
TBD based on the above conversation (https://github.com/omry/omegaconf/pull/465#discussion_r546915451)